PySoar/generalFunctions.py

```diff
@@ -213,9 +213,9 @@ def task_url_from_daily(daily_url):
 
 
 def url_format_correct(url_string):
-    if url_string[0:26] != "http://www.soaringspot.com":
+    if (url_string[0:26] != "http://www.soaringspot.com" and url_string[0:21] != "http://www.strepla.de") :        
         return 'URL should start with http://www.soaringspot.com'
```

Comment:
Yes, and it might also be a good idea to change the description for the entry widget in main_pysoar.py to something like: "Give Soaringspot/scoringStrepla URL"